Source/Libraries/GSF.Core/Data/DataInserter.cs

```diff
@@ -616,7 +615,7 @@ private void ExecuteInserts(Table fromTable, Table toTable)
                     case DatabaseType.SQLServer:
                         try
                         {
-                            toTable.Connection.ExecuteNonQuery("SET IDENTITY_INSERT " + toTable.SQLEscapedName + " ON", Timeout);
+                            toTable.Connection.ExecuteNonQuery("SET IDENTITY_INSERT {0} ON", toTable.SQLEscapedName, Timeout);
```

Comment:
Parameter substitution does not work on Tables

Reply:
updated